### ISSUE TITLE
Core: Tooltips not Announced if Speech Output Paused with the Shift Key from within Windows Explorer

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#10680]
### Steps to reproduce:

1. Ensure that NVDA is set to announce tooltips

2. From an Explorer window, place focus on a file/folder and ensure that tooltips are being output by NVDA as expected

3. Place the Windows cursor on a separate file/folder, and whilst NVDA is speaking its name hit either shift key to pause output - press the shift key again after output has been paused to resume output

### Actual behavior:

You'll notice that after resuming output with the shift key, NVDA then fails to announce the tooltip - as would usually be done if speech was not interupted

### Expected behavior:

NVDA should continue to read tooltips as normal after output is resumed

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2019.2.1

#### Windows version:

Windows 7 X64

**Comment by Adriani90** [comment]
Actually this is not expected because tooltips appear and disappear after a short period. So NVDA does not save the tooltip message in any buffer. This is also ok because a sighted person is not able to save a tooltip either. I suggest closing this issue as invalid.
cc: @lukaszgo1, @josephsl

**Comment by Adriani90** [comment]
Closing as invalid.
